src/site_apportionment/site_apportionment.py

```diff
@@ -200,34 +211,37 @@ def create_category_df(df: pd.DataFrame, value_cols: List[str]) -> pd.DataFrame:
     category_df = keep_good_markers(category_df)
 
     # De-duplicate - possibly, not needed
```

Comment:
# Category_df will contain textual columns that belong to the responses, these will be included in the list below.
# Note that the list textual_cols also includes the numeric 9000 cols not needed for imputation, and the columns
# from the contributor data, originating in IDBR.